src/test_helper.rs

```diff
@@ -208,26 +205,24 @@ impl Widget for DebugWidget {
 /// dumped to stdout.
 ///
 /// ```sh
-/// DEBUG_SNAPSHOT=1 cargo test --lib -- --nocapture --test simple_block_test
+/// DEBUG_SNAPSHOT=1 cargo test --lib -- --show-output --test simple_block_test
 /// ```
 ///
 /// !!! The normal test output frequently interferes which results in scrambled output, especially
 /// when multiple tests are run at once.
 /// Running it multiple times might usually leads to good output (for now, with small widget output)
 pub fn print_buffer(buffer: &Buffer) -> std::io::Result<()> {
     if env::var("DEBUG_SNAPSHOT").is_ok() {
-        enable_raw_mode()?;
-        execute!(stdout(), MoveToNextLine(0))?;
         let mut terminal = Terminal::with_options(
             CrosstermBackend::new(stdout()),
             TerminalOptions {
-                viewport: Viewport::Inline(buffer.area.height),
+                viewport: Viewport::Fixed(buffer.area),
             },
         )?;
+
+        terminal.clear()?;
         terminal.current_buffer_mut().clone_from(buffer);
         terminal.flush()?;
```

Comment:
You can add the following to get to the beginning of the next line:

```suggestion
        terminal.flush()?;
        crossterm::queue!(stdout(), crossterm::cursor::MoveTo(0, buffer.area.height))?;
```

Reply:
Sounds good. I'll try.